From: Felix Fietkau <nbd@nbd.name>
Date: Mon, 18 Jan 2016 12:27:49 +0100
Subject: [PATCH] Kbuild: don't hardcode path to awk in scripts/ld-version.sh

On some systems /usr/bin/awk does not exist, or is broken. Find it via
$PATH instead.

Signed-off-by: Felix Fietkau <nbd@nbd.name>
---

--- a/scripts/ld-version.sh
+++ b/scripts/ld-version.sh
@@ -1,5 +1,6 @@
-#!/usr/bin/awk -f
+#!/bin/sh
 # extract linker version number from stdin and turn into single number
+exec awk '
 	{
 	gsub(".*\\)", "");
 	gsub(".*version ", "");
@@ -8,3 +9,4 @@
 	print a[1]*100000000 + a[2]*1000000 + a[3]*10000;
 	exit
 	}
+'
